lib/holdings/item.rb

```diff
@@ -134,8 +133,17 @@ def stackmapable?
       stackmapable_library? && stackmapable_location?
     end
 
+    def bound_with_parent
+      return nil unless document && document['holdings_json_struct']
```

Comment:
```suggestion
      return nil unless document && document['holdings_json_struct'].any?
```

I think an empty array would also evaluate to `true` and break with a NoMethod error below, right?